actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -846,12 +846,13 @@ def select_year
           options[:step]              = options[:start] < options[:end] ? 1 : -1
           options[:leading_zeros]     = false
           options[:max_years_allowed] = @options[:max_years_allowed] || 1000
+          options[:labels_for_year_options]    = @options[:labels_for_year_options]
```

Comment:
I think this line was tagged by code climate. I _think_ we stopped aligning operators, so you can just leave 1 space before `=`.